src/NUnitFramework/framework/Internal/TestParameters.cs

```diff
@@ -32,23 +34,29 @@ namespace NUnit.Framework.Internal
     /// </summary>
     public abstract class TestParameters : ITestData, IApplyToTest
     {
+        internal static readonly object[] NoArguments =
+#if NET35 || NET40 || NET45 // New in net46
+            new object[0];
+#else
+            Array.Empty<object>();
```

Comment:
I like the new syntax, but it it over complicating things with the `#if`. What advantage does the net46 syntax gain us here since  `NoArguments` is static?

Reply:
I added it as a reminder to come back and fix one day, but there is no advantage I can think of besides the habit. The complication isn't nice.

Reply:
This is already in master too. Shall I revert in this PR?

Reply:
No worries if it is already there...